Source/ASImageNode+AnimatedImage.mm

```diff
@@ -53,6 +53,14 @@ - (void)_locked_setAnimatedImage:(id <ASAnimatedImageProtocol>)animatedImage
   if (ASObjectIsEqual(_animatedImage, animatedImage)) {
     return;
   }
+    
+  if (animatedImage == nil) {
```

Comment:
I don't understand this check. Shouldn't we do this whenever `_animatedImage != nil`?

Reply:
Yeeeeees, good catch.